cpp/cmake/Dependencies.cmake

```diff
@@ -39,7 +39,7 @@ else(DEFINED ENV{RAFT_PATH})
 
   ExternalProject_Add(raft
     GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           37faba16f263b6556583208517d4237d9041bfc0
+    GIT_TAG           d4a19c6328dbdf43666926249a7b676de7ad5606
```

Comment:
Ah I see what is the issue here, this commit hash corresponds to the latest commit in the fork/PR of RAFT, but since we switched to squashing the commits, at the moment of merging this changed to https://github.com/rapidsai/raft/commit/7d83f0ddd2255d9de51457ae76901fbc5cbf96e5. That said, I think we should update it to the latest (6272ef1073e7d600e6d49588846e916155b9b245)

Reply:
That makes sense! Just committed the fix. Hopefully everything passes!